installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -5,6 +5,14 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+In environments where the cloud identity and access management (IAM) APIs are not reachable, or the administrator prefers not to store an administrator-level credential secret in the cluster `kube-system` namespace, you can put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
```

Comment:
What is the purpose of this sentence? It is a repeat of one of the options in the very next section.

Reply:
It is a short introduction to the whole assembly / document.